apps/web/src/scenes/NftDetailPage/NftDetailAsset.tsx

```diff
@@ -28,6 +32,8 @@ type NftDetailAssetComponentProps = {
   onLoad: () => void;
 };
 
+const DESKTOP_TOKEN_SIZE = 600;
```

Comment:
1. noticed this is defined in another file as well; can we define it one place and export it?
2. probably need mobile support as well

Reply:
yess done now